### PR TITLE
Healthcheck functionality through Spring Boot actuator

### PR DESCRIPTION
**Pull request by aComanche**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,157 @@
...
+.idea/httpRequests
+
+# Android studio 3.1+ serialized cache file
+.idea/caches/build_file_checksums.ser
+
+### Intellij+all Patch ###
+# Ignores the whole .idea folder and all .iml files
+# See https://github.com/joeblau/gitignore.io/issues/186 and https://github.com/joeblau/gitignore.io/issues/360
+
+.idea/
```
Comment (gitmik) [inline .gitignore]: since we ignore whole .idea folder, do we need above rules for specific files in .idea/ path ?

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,157 @@
-# https://github.com/github/gitignore/blob/master/Maven.gitignore
+# Created by https://www.gitignore.io/api/java,linux,macos,maven,windows,intellij+all
```
Comment (gitmik) [inline .gitignore]: commit msg says make it more concise and useful, but it's now almost 200 lines
concise is a synonym for brief and compact :) 
can we have this at least brief and compact ?

File: tcp-mocker-web/src/main/java/io/payworks/labs/tcpmocker/TCPMockerApplication.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package io.payworks.labs.tcpmocker;
```
Comment (gitmik) [inline tcp-mocker-web/src/main/java/io/payworks/labs/tcpmocker/TCPMockerApplication.java]: There is already a starter class io.payworks.labs.tcpmocker.MockServerApp
does this one serve other purposes?

https://github.com/payworks/tcp-mocker/blob/master/tcp-mocker-app/src/main/java/io/payworks/labs/tcpmocker/MockServerApp.java

File: .gitignore (hunk 2 above)
Comment (andreysaksonov) [reply .gitignore]: Please do not add here files, which are not used explicitly in repo (i.e. maven & docker). Usage of intellij is not mandatory for this project. Such exclusions should be in n user-level gitignore, not in project-level.

File: tcp-mocker-web/src/main/java/io/payworks/labs/tcpmocker/TCPMockerApplication.java (hunk 3 above)
Comment (aComanche) [reply tcp-mocker-web/src/main/java/io/payworks/labs/tcpmocker/TCPMockerApplication.java]: I think it does, that class is to handle the tcp server and this other would be to access the tcp mocker microservice, like the recordings controller

**Pull request closed without merging**